package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/etag",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

Comment:
Please don't change the version, it will be done by the releaser.

Reply:
```suggestion
  "version": "5.0.0",
```